bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx99-de.html

```diff
@@ -0,0 +1,46 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ #}
+
+{% from "macros-protocol.html" import card with context %}
+
+{% extends "firefox/whatsnew/base.html" %}
+
+{% block page_title %}{{ ftl('whatsnew-page-title') }}{% endblock %}
+{% block page_desc %}{{ ftl('whatsnew-page-description') }}{% endblock %}
+
+{% block body_id %}firefox-whatsnew{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_99_eu') }}
+{% endblock %}
+
+{% set _entrypoint = 'mozilla.org-whatsnew99' %}
+{% set utm_campaign = campaign + '-fr' %}
```

Comment:
Issue: should this be `de`?

```suggestion
{% set utm_campaign = campaign + '-de' %}
```